source/_integrations/vicare.markdown

```diff
@@ -115,15 +144,7 @@ Eco mode reduces the target temperature by 3°C, whereas Comfort mode sets the t
 | `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use `all` keyword instead of entity_id.
 | `preset_mode` | no | New value of preset mode.
```

Comment:
```suggestion
| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use `all` keyword instead of entity_id. |
| `preset_mode` | no | New value of preset mode. |
```